source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -28,5 +27,78 @@ more naturally readable.
 Operations
 ----------
 
+The following sections provide information and examples for
+aggregation expression operations available in the driver.
+The operations are categorized by purpose and functionality.
+
+.. important::
+
+   The driver generates a Query API expression that may be different
+   from the Query API expression provided in each example. However,
+   the expressions are equivalent when evaluated.
+
+Boolean Operations
+~~~~~~~~~~~~~~~~~~
+
+You can construct a boolean operation from at least one value of type
+``MqlBoolean`` and one or more boolean aggregation methods.
+
+The following table describes boolean aggregation expression operators
+available in the driver and equivalent expressions in the
+Query API. The methods link to API documentation and the Query API
+expressions link to Server manual documentation.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Aggregation Operation
+     - Query API Expression
```

Comment:
```suggestion
   * - Java Method
     - Aggregation Pipeline Operator
```
Trying to simplify / avoid introducing new terms.

Since we're just listing methods, and the context is clear enough, so I don't think we need qualifiers (and "aggregation method" sounds odd or misleading). For "Query API Expression", unsure if any docs call `$not` an expression (it isn't one, formally), but in the docs all of [these pages](https://www.mongodb.com/docs/manual/reference/operator/aggregation/) are under the [Aggregation Pipeline Operator](https://www.mongodb.com/docs/manual/reference/operator/aggregation/) page, which seems like a fine enough term.